fineract-branch/src/main/java/org/apache/fineract/organisation/teller/domain/model/request/CashierRequest.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.organisation.teller.domain.model.request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import java.io.Serial;
+import java.io.Serializable;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.SneakyThrows;
+
+@Data
+@NoArgsConstructor
+public class CashierRequest implements Serializable {
+
+    @Serial
+    private static final long serialVersionUID = 1L;
+
+    @Size(max = 100)
+    private String description;
+    @NotNull
+    private Boolean isFullDay;
+    @NotNull
+    @Min(value = 1)
+    private Long staffId;
+    @NotNull
+    private String dateFormat;
+    @NotNull
+    private String startDate;
+    @NotNull
+    private String endDate;
+    private String locale;
+    private String hourStartTime;
+    private String minStartTime;
+    private String hourEndTime;
+    private String minEndTime;
+
+    @SneakyThrows
+    public String toJson() {
+        return new ObjectMapper().setSerializationInclusion(JsonInclude.Include.ALWAYS).writeValueAsString(this);
```

Comment:
Why not using the proper JSON serializer that we have been using? (That's preconfigured)